### ISSUE TITLE
Missing lint errors for a few cases (E713)

### ISSUE THREAD

**Issue by asottile** [PyCQA/pycodestyle#583]
This should flag three things but only flags one (code is changed a bit from the original problem for demonstration purposes)

``` python
def f(x, y):
    return x[y]


ns = {'a': 1, 'y': (2, 3)}

if not f(ns, 'a') in f(ns, 'y'):
    print('ohai')

if not ns['a'] in ns['y']:
    print('ohai2')

if not 'a' in 'foo':
    print('ohai3')
```

**Comment by sigmavirus24** [comment]
So I ran pycodestyle against this because I wasn't sure what was missing. The output was:

```
ex.py:13:4: E713 test for membership should be 'not in'
```

That helped me figure out the other two items that should have been caught. Only the last `if not ... in ...` was caught. I'll try to figure out the other two cases. Thanks for the report @asottile

**Comment by sigmavirus24** [comment]
So I have several test cases that demonstrate this behaviour. The problem is now determining how to rewrite that regular expression without breaking the old behaviour.

**Comment by asottile** [comment]
Duplicate of #334
